src/styles/blocks/header.scss

```diff
@@ -0,0 +1,61 @@
+@use "../utils/mixins.scss" as underLine;
+@use "../utils/variables" as mainColor;
+
+.header {
+  display: flex;
+  justify-content: space-between;
+  padding-left: 50px;
+  padding-right: 50px;
+  height: 60px;
+  box-shadow: 0 2px 4px rgba(0, 0, 0, 0.05);
+}
+
+.logo-site {
+  height: 40px;
+  margin-top: 10px;
+}
+
+.nav {
+  display: flex;
+  align-items: center;
+
+  &__list {
+    list-style: none;
+    display: flex;
+    padding: 0;
+  }
+  
+  &__item {
+    margin-right: 20px;
+    text-transform: uppercase;
+    font-size: 12px;
+    letter-spacing: 0;
+    font-weight: 500;
+    
+    &:nth-last-child(1) {
+      margin-right: 0;
+    }
+
+    & .is-active {
+      color: mainColor.$main-link-color;
+    }
+  }
+  
+  .nav__link {
+    display: flex;
+    height: 60px;
+    justify-content: center;
+    align-items: center;
+    position: relative;
```

Comment:
Pay attention to your css properties order
https://9elements.com/css-rule-order/